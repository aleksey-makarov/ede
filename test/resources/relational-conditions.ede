{ "one": 1, "ten": 10 }
---
{% if one %}
{{ one }}
{% endif %}

{% if 1 <= 10 %}
1 <= 10
{% else %}
nein. :(
{% endif %}

{% if 8 <= ten %}
8 <= {{ ten }}
{% else %}
oh lordy nein.
{% endif %}

{% if one <= 2 %}
{{ one }} <= 2
{% else %}
ack.
{% endif %}

{% if one == ten %}
No..
{% else %}
{{ one }} != {{ ten }}
{% endif %}

{% if ten != one %}
{{ ten }} != {{ one }}
{% endif %}

{% if ten > one %}
{{ ten }} > {{ one }}
{% endif %}

{% if ten >= ten %}
{{ ten }} >= {{ ten }}
{% endif %}

{% if one <= one %}
{{ one }} <= {{ one }}
{% endif %}

{% if ten == 10 %}
{{ ten }} == 10
{% endif %}
