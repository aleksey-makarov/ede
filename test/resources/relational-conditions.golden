1

1 <= 10

8 <= 10

1 <= 2

1 /= 10

10 /= 1

10 > 1

10 >= 10

1 <= 1

10 == 10
